src/main/java/org/folio/circulation/services/EventPublisher.java

```diff
@@ -54,8 +56,10 @@ public EventPublisher(RoutingContext routingContext) {
       write(payloadJsonObject, LOAN_ID_FIELD, loan.getId());
       write(payloadJsonObject, DUE_DATE_FIELD, loan.getDueDate());
 
-      return pubSubPublishingService.publishEvent(
-        ITEM_CHECKED_OUT.name(), payloadJsonObject.encode())
+      JsonObject logEventPayload = mapToCheckOutLogEventJson(loanAndRelatedRecords);
+      CompletableFuture.runAsync(() -> pubSubPublishingService.publishEvent(LOG_RECORD_EVENT.name(), logEventPayload.encode()));
```

Comment:
Does this mean that the custom log message publishing code is no longer used?

Reply:
@VasilyGan @marcjohnson-kint @TarasSpashchenko could you please clarify what is a final decision? custom library https://github.com/folio-org/mod-audit/blob/master/mod-audit-client/src/main/java/org/folio/util/PubSubLogPublisherUtil.java or existing circulation Pubsub publisher. This question blocks my current implementation and all the feature. Thanks!

Reply:
@khandramai, as we discussed during today's call, a custom library should be used to abstract away PubSub and provide the ability to easily switch to another Kafka client library in the future.

Reply:
When this code is changed, please can comments be included in the code to explain the rationale of using two different  publishing approaches